doc/source/coding-style/required-standard.rst

```diff
@@ -36,9 +35,9 @@ Required ``pyproject.toml`` configuration
     [tool.pydocstyle]
     convention = "numpy"
 
-
 Required ``.flake8`` configuration
 ----------------------------------
+
 The following ``.flake8`` file is also required:
```

Comment:
```suggestion
The following ``.flake8`` file is also required:
```
Is the name of the file ``.flake8``? Of so, leave what's there. If it's a general reference, rewrite to "The following Flake8 file ,,,,"

Reply:
Yes. Its name of the file.